src/pages/Lend/Filter/FilterSections/LoanDetails/RangeSliderWrapper.vue

```diff
@@ -1,29 +1,22 @@
 <template>
 	<ais-range-input
 		:attribute="queryAttribute"
-		:min="minimumValue"
-		:max="maximumValue"
```

Comment:
This is what fixed the US loan bug, the rest of the PR was to support the dynamic min and max that this results in